src/components/DetailViewHeader.jsx

```diff
@@ -37,26 +49,38 @@ export class DetailViewHeader extends React.PureComponent {
     addSteward: PropTypes.func,
     canReadPolicies: PropTypes.bool,
     classes: PropTypes.object.isRequired,
+    dispatch: PropTypes.func.isRequired,
+    isProcessing: PropTypes.bool,
+    isDone: PropTypes.bool,
     of: PropTypes.object,
+    exportResponse: PropTypes.object,
     readers: PropTypes.arrayOf(PropTypes.string),
     removeReader: PropTypes.func,
     terraUrl: PropTypes.string,
     removeSteward: PropTypes.func,
     stewards: PropTypes.arrayOf(PropTypes.string).isRequired,
   };
 
+  exportToWorkspaceCopy = () => {
+    const { dispatch, of } = this.props;
+    dispatch(exportSnapshot(of.id));
```

Comment:
We should test timing here.  But for clicks that launch jobs, we should consider having something to track _just_ that the user has clicked to ensure that the user doesn't click again while the job is starting (maybe that could get handled inside the `exportSnapshot` action reducer)

Reply:
It would be pretty easy for the click handler to set the element to `disabled`, and then have the success/failure action un-disable it.

Reply:
Good point @tlangs  - I tried this, but ran into a problem with this approach when the export encountered an exception. Since we're handling the exception in redux, I just added a variable to that action, as Muscles suggested. Although, as I mentioned before, we're already hiding the button using the same action, so I don't really think disabling it is doing much, if anything.